src/rgw/rgw_iam_policy.h

```diff
@@ -363,6 +363,13 @@ struct Condition {
     }
   };
 
+  struct ci_starts_with {
```

Comment:
It's fine as-written, but you could also just make this an ordinary function, since it's not tracking any data.